package.json

```diff
@@ -37,13 +37,14 @@
     "truffle": "^4.0.1",
     "truffle-assertions": "^0.7.1",
     "truffle-flattener": "^1.2.5",
-    "web3": "^0.20.7",
-    "web3-utils": "^1.2.1",
+    "web3": "^1.2.4",
+    "web3-utils": "^1.2.4",
     "yargs": "^11.0.0"
   },
   "devDependencies": {
     "@nomiclabs/buidler": "^1.0.1",
     "chai": "^4.1.2",
+    "chai-bn": "^0.2.0",
```

Comment:
do we need chai-bn in separate package
though it was part of the environment...